python/src/BernsteinCopulaFactory_doc.i.in

```diff
@@ -26,24 +26,32 @@ DistributionFactory, EmpiricalBernsteinCopula"
 
 Parameters
 ----------
-sample : 2-d sequence of float, of dimension *d*
-    The sample of size :math:`n>0` from which the copula is estimated.
+sample : 2-d sequence of float, of dimension :math:`d`
+    The sample of size :math:`\sampleSize>0` from which the copula is estimated.
 method : str
     The name of the bin number selection method. Possible choices are *AMISE*, *LogLikelihood* and *PenalizedCsiszarDivergence*. Default is *LogLikelihood*, given by the *'BernsteinCopulaFactory-BinNumberSelection'* entry of :class:`~openturns.ResourceMap`.
 m : int
     The number of sub-intervals in which all the edges of the unit cube
     :math:`[0, 1]^d` are regularly partitioned.
+    
+    Default value is 1.
 
 Returns
 -------
-copula : :class:`~openturns.Distribution`
-    The estimated copula as a generic distribution.
+core : :class:`~openturns.Distribution`
+    The empirical Bernstein copula as a generic distribution.
 
+Notes
+-----
+Let :math:`\sampleSize` be the size of the sample. If :math:`m` divides :math:`\sampleSize`, then the
+:class:`~openturns.EmpiricalBernsteinCopula` built from the sample is a 
+copula. Otherwise, the :class:`~openturns.EmpiricalBernsteinCopula` built from the sample is a core, i.e a
+multivariate distribution whose range is included in :math:`[0,1]^d`.
```

Comment:
The difference between a core and a distribution is rather implicit in this text.
```suggestion
multivariate distribution whose range is included in :math:`[0,1]^d` 
(but the marginals are not necessarily uniform) because some cells may
have more points than others.
```
It is suggested to create a new theory help which introduces this difference more clearly.

Reply:
Yes... actually, the methods build of the class ensure that the result is a copula. So i removed that comment. I detailed that in the EmpiricalBernsteinCopula class.